build/appprotect/DockerfileWithAppProtectForPlusForOpenShift

```diff
@@ -83,13 +83,13 @@ RUN  mkdir -p /var/lib/nginx \
 	&& mkdir -p /var/log/app_protect \
 	&& mkdir -p /opt/app_protect \
 	&& chown -R nginx:0 /etc/app_protect \
-	&& chown -R nginx:0 /usr/share/ts \
-	&& chown -R nginx:0 /etc/nginx \
-	&& chown -R nginx:0 /var/cache/nginx \
-	&& chown -R nginx:0 /var/lib/nginx/ \
-	&& chown -R nginx:0 /var/log/app_protect/ \
-	&& chown -R nginx:0 /opt/app_protect/ \
-	&& chown -R nginx:0 /var/log/nginx/ \
+	                    /etc/nginx \
```

Comment:
the order of folders here is a bit different from the file `build/appprotect/DockerfileWithAppProtectForPlus`. 
Could we use the same order for consistency?

Reply:
@pleshakov I've sorted that out cause in that order it has better readility, however I can revert that one.